src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs

```diff
@@ -41,7 +41,7 @@ public GtkHost(Func<WinUIApplication> appBuilder)
 
 		internal static GtkHost? Current => _current;
 
-		internal UnoGtkWindow? MainWindow { get; private set; }
+		internal UnoGtkWindow? MainWindow { get; set; }
```

Comment:
Provide a way to enumerate open windows and access the native window on each target.